qa/L0_sequence_stress/sequence_stress.py

```diff
@@ -397,6 +397,13 @@ def stress_thread(name, seed, pass_cnt, correlation_id_base, trial, model_name,
                                    sequence_name=name)
                     last_choices[client_idx] = "valid"
 
+        # We need to explicitly close each client so that streams get
+        # cleaned up and closed correctly, otherwise the application
+        # can hang when exiting.
+        for c, i in client_metadata_list:
+            print("thread {} closing client {}".format(name, i))
+            c.close()
+
     except Exception as ex:
         _thread_exceptions_mutex.acquire()
```

Comment:
Can you also close the client in the exception handler?

Reply:
Done